languages/python/docs.nix

```diff
@@ -80,6 +80,7 @@ in
 
       mkdir -p doc-source
       echo "" >> doc-source/conf.py # generated conf.py does not end in a newline
+      echo "extensions = ['sphinx.ext.napoleon']" >> doc-source/conf.py # support google doc strings
```

Comment:
Nice!

How about also adding
`echo "napoleon_include_init_with_doc = True" >> doc-source/conf.py` ?
Otherwise the init is mostly undocumented (you just see the name and type of the arguments, no docstring text).

Also it seems like `[source]` links disappears (at least with the rtd theme), haven't seen any config value that should affect it.

Reply:
It murdered the default extensions (extensions = [] was defined twice)